### DISCUSSION TITLE
Analyzing the GitHub marketplace - Dependency security is a big issue

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#17] [Comments]
# Analyzing the GitHub marketplace - Dependency security is a big issue

Analyzing the GitHub marketplace - Dependency security is a big issue

https://devopsjournal.io/blog/2022/09/18/Analysing-the-GitHub-marketplace

**Comment by PavanMudigonda** [comment]
Hello, 

I request you to delete repo https://github.com/actions-marketplace-validations/PavanMudigonda_jacoco-reporter/issues/1

PLEASE DELETE THIS REPO. ITS SHOWING WRONG METRICS for OUR USERS.
